### DISCUSSION TITLE
Is it possible to assign a color to the color picker on load (without the use of inline mode?)

### DISCUSSION THREAD

**Discussion by Potato22** [mdbassit/Coloris#165] [Q&A] [answered]
I intend to assign a random color on load to the color pickers but haven't figured out a good way to do it properly, let me know if there's something I should know or missed.

**Comment by mdbassit** [comment, marked as answer]
The color picker will automatically take the value of the field that opens. See the examples here:
https://coloris.js.org/examples.html

**Reply by Potato22** [reply]
I see, it uses the input field as the initiator. Seems like I will have to modify the input content programmatically on load instead, thanks.
